src/pkg/backup/details/details.go

```diff
@@ -1025,6 +1026,50 @@ func (i *OneDriveInfo) updateFolder(f *FolderInfo) error {
 	return updateFolderWithinDrive(OneDriveItem, i.DriveName, i.DriveID, f)
 }
 
+// GroupsInfo describes a groups item
+type GroupsInfo struct {
+	Created    time.Time `json:"created,omitempty"`
+	DriveName  string    `json:"driveName,omitempty"`
+	DriveID    string    `json:"driveID,omitempty"`
+	ItemName   string    `json:"itemName,omitempty"`
+	ItemType   ItemType  `json:"itemType,omitempty"`
+	Modified   time.Time `json:"modified,omitempty"`
+	Owner      string    `json:"owner,omitempty"`
+	ParentPath string    `json:"parentPath,omitempty"`
+	Size       int64     `json:"size,omitempty"`
+}
+
+// Headers returns the human-readable names of properties in a SharePointInfo
+// for printing out to a terminal in a columnar display.
+func (i GroupsInfo) Headers() []string {
+	return []string{"Created", "Modified"}
+}
+
+// Values returns the values matching the Headers list for printing
+// out to a terminal in a columnar display.
+func (i GroupsInfo) Values() []string {
+	return []string{
+		dttm.FormatToTabularDisplay(i.Created),
+		dttm.FormatToTabularDisplay(i.Modified),
+	}
+}
+
+func (i *GroupsInfo) UpdateParentPath(newLocPath *path.Builder) {
+	i.ParentPath = newLocPath.PopFront().String()
+}
+
+func (i *GroupsInfo) uniqueLocation(baseLoc *path.Builder) (*uniqueLoc, error) {
+	return nil, clues.New("not yet implemented")
+}
+
+func (i *GroupsInfo) updateFolder(f *FolderInfo) error {
```

Comment:
We can maybe add it to [`updateFolder`](https://github.com/alcionai/corso/blob/9e655e029cdbd300beddbbeffbb6512fd81e9cf3/src/pkg/backup/details/details.go#L788). Same for `uniqueLocation`.

Reply:
I think I got this in a follow-up PR.